infra/README.md

```diff
@@ -214,14 +233,15 @@ You can now install Hail:
   export HAIL_BUILDKIT_IMAGE=$HAIL_DOCKER_PREFIX/hail-buildkit:cache
   export BATCH_WORKER_IMAGE=$HAIL_DOCKER_PREFIX/batch-worker:cache
   export HAIL_CI_BUCKET_NAME=dummy
+  export HAIL_TEST_GCS_BUCKET=<hail-test-gcs-bucket>
```

Comment:
Do you need the new `HAIL_TEST_REQUESTER_PAYS_GCS_BUCKET`?

Reply:
I actually think some of these environment variables are unnecessary. I'll make a todo in asana to remove the ones that are no longer needed.